Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -231,6 +231,7 @@
 #include <WebCore/HTMLModelElement.h>
 #include <WebCore/HTMLPlugInElement.h>
 #include <WebCore/HTMLSelectElement.h>
+#include <WebCore/HTMLTextAreaElement.h>
```

Comment:
Nit: This could be limited to the `#if PLATFORM(IOS_FAMILY)` block of includes, but I don't think that's important.